portfolio/src/test/java/com/google/sps/ShortestTaskFirstTest.java

```diff
@@ -0,0 +1,187 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ShortestTaskFirstTest {
+  private static Instant BEGINNING_OF_DAY = Instant.parse("2020-06-25T00:00:00Z");
+  private static Instant END_OF_DAY = Instant.parse("2020-06-25T23:59:59Z");
+  private static Instant THREE_DAYS_LATER = Instant.parse("2020-06-28T00:00:00Z");
+
+  private static Duration TIME_15_MINUTES = Duration.ofSeconds(900);
+  private static Duration TIME_20_MINUTES = Duration.ofSeconds(1200);
+  private static Duration TIME_30_MINUTES = Duration.ofSeconds(1800);
+  private static Duration TIME_45_MINUTES = Duration.ofSeconds(2700);
+  private static Duration TIME_60_MINUTES = Duration.ofSeconds(3600);
+
+  private static Instant TIME_0900 = Instant.parse("2020-06-25T09:00:00Z");
+  private static Instant TIME_0920 = TIME_0900.plus(TIME_20_MINUTES);
+  private static Instant TIME_0950 = TIME_0900.plus(Duration.ofSeconds(3000));
+  private static Instant TIME_1000 = TIME_0900.plus(TIME_60_MINUTES);
+  private static Instant TIME_1020 = TIME_1000.plus(TIME_20_MINUTES);
+  private static Instant TIME_1030 = TIME_1000.plus(TIME_30_MINUTES);
+  private static Instant TIME_1100 = TIME_1000.plus(TIME_60_MINUTES);
+  private static Instant TIME_1200 = TIME_1100.plus(TIME_60_MINUTES);
+  private static Instant TIME_1300 = TIME_1200.plus(TIME_60_MINUTES);
+  private static Instant TIME_1400 = TIME_1300.plus(TIME_60_MINUTES);
+  private static Instant TIME_1500 = TIME_1400.plus(TIME_60_MINUTES);
+  private static Instant TIME_1600 = TIME_1500.plus(TIME_60_MINUTES);
+  private static Instant TIME_1700 = TIME_1600.plus(TIME_60_MINUTES);
+  private static Instant TIME_1800 = TIME_1700.plus(TIME_60_MINUTES);
+
+  private final TaskPriority LOW_PRIORITY = new TaskPriority(1);
+
+  // Makes sure we return an empty list in the case where no tasks are passed.
+  @Test
+  public void noTasksScheduled() {
+    ShortestTaskFirst algorithm = new ShortestTaskFirst();
```

Comment:
Consider renaming the variable to shortestTaskFirstScheduler. I think algorithm is a vague name.

Reply:
Done.